cmds/jrsonnet/Cargo.toml

```
@@ -13,22 +13,23 @@ mimalloc = ["mimallocator"]
 # Experimental feature, which allows to preserve order of object fields
 exp-preserve-order = [
     "jrsonnet-evaluator/exp-preserve-order",
-    "jrsonnet-evaluator/exp-serde-preserve-order",
     "jrsonnet-cli/exp-preserve-order",
-    "jrsonnet-cli/exp-serde-preserve-order",
 ]
 # Destructuring of locals
 exp-destruct = ["jrsonnet-evaluator/exp-destruct"]
+# Iteration over objects yields [key, value] elements
+exp-object-iteration = ["jrsonnet-evaluator/exp-object-iteration"]
+
 # std.thisFile support
 legacy-this-file = ["jrsonnet-cli/legacy-this-file"]
 
 [dependencies]
-jrsonnet-evaluator = { path = "../../crates/jrsonnet-evaluator", version = "0.4.2" }
-jrsonnet-parser = { path = "../../crates/jrsonnet-parser", version = "0.4.2" }
-jrsonnet-cli = { path = "../../crates/jrsonnet-cli", version = "0.4.2" }
+jrsonnet-evaluator.workspace = true
+jrsonnet-parser.workspace = true
+jrsonnet-cli.workspace = true
 jrsonnet-gcmodule = { version = "0.3.4" }
 
 mimallocator = { version = "0.1.3", optional = true }
 thiserror = "1.0"
-clap = { version = "3.2", features = ["derive"] }
-clap_complete = { version = "3.2" }
+clap = { version = "4.0", features = ["derive"] }
+clap_complete = { version = "4.0" }
```

Stick to exact versions:

```suggestion
thiserror = "1.0.37"
clap = { version = "4.0.23", features = ["derive"] }
clap_complete = { version = "4.0.5" }
```

---

I expect dependencies to maintain semver, and using not exact versions allows easier usage of jrsonnet as a library